### DISCUSSION TITLE
Unraid Documentation with a reverse proxy already running (swag, NPM, or traefik).

### DISCUSSION THREAD

**Discussion by taos15** [nextcloud/all-in-one#2553] [Questions] [answered]
### Is your feature request related to a problem? Please describe. 
- I have a hard time converting the run commands to unraid.
- The container will always start with the `AIO is currently in "reverse proxy mode"` notification.

### Describe the solution you'd like
- It would help mention of setting it up in unraid as well as what to put to not use the internal reverse proxy.

### Additional context
- I run treafik in unraid
- I have a instance of nextcloud that I want to migrate to AIO
- I dont want AOI to do anything with the crts since I already have a route and ssl for it
- i want to be able to change the config valume to match unraid `/mnt/user/appdata/APPNAME`

**Comment by szaimen** [comment, marked as answer]
> * The container will always start with the `AIO is currently in "reverse proxy mode"`

Did you read any further?

> AIO is currently in "reverse proxy mode" which means that it can be installed behind a web server or reverse proxy (like Apache Nginx and else) and does not do the TLS proxying itself.

---

> * I run treafik in unraid

Traefik is covered by https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md

> * I have a instance of nextcloud that I want to migrate to AIO

See https://github.com/nextcloud/all-in-one/blob/main/migration.md

> * I dont want AOI to do anything with the crts since I already have a route and ssl for it

Also see https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md

> * i want to be able to change the config valume to match unraid `/mnt/user/appdata/APPNAME`

That is not possible by design as the built-in backup solution would not work in that case anymore. However see https://github.com/nextcloud/all-in-one#how-to-change-the-default-location-of-nextclouds-datadir

**Comment by taos15** [comment]
Thanks. I will give be another shot.
